java/ql/lib/semmle/code/java/security/performance/RegExpTreeView.qll

```diff
@@ -0,0 +1,57 @@
+/**
+ * This module should provide a class hierarchy corresponding to a parse tree of regular expressions.
+ */
+
+import java
+import semmle.code.java.regex.RegexTreeView
```

Comment:
It is quite confusing to have both a `RegexTreeView.qll` and a `RegExpTreeView.qll`. Could we improve on that? It took me a few look-throughs to figure out that they were two different files.